src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/0.23.0/fastqc-version.xml

```diff
@@ -6,12 +6,9 @@
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
     <changeSet id="file-processing-state" author="tom">
         <addColumn tableName="analysis_fastqc">
-            <column name="fastqcVersion" type="varchar(255)"/>
+            <column name="fastqcVersion" type="varchar(255)" value="0.10.1">
```

Comment:
Could you maybe merge `development` into here and in the cart base branch, since I think this change is just an artefact of not having the most recent code merged in.

Reply:
Sure, that was a little behind wasn't it!  Done.